src/client/tile.cpp

```diff
@@ -1175,7 +1175,18 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 					core::rect<s32>(pos_from, dim),
 					video::SColor(255,255,255,255),
 					NULL);*/
-			blit_with_alpha(image, baseimg, pos_from, pos_to, dim);
+
+			v2u32 dim_dst = baseimg->getDimension();
```

Comment:
I agree but `v2u32` is much shorter, thus easier to read and should be used in the code above. Is there somewhere a code rule about this?